docs/pages/project/contributing.md

```diff
@@ -35,13 +35,13 @@ Relevant coding style guidelines are the Go Code Review Comments and the Formatt
 
 ## <a name="contributing">General Contribution Flow</a>
 
-In order to contribute to Meshery, please follow the fork-and-pull request workflow described [here](./git-workflow.md).
+To contribute to Meshery, please follow the fork-and-pull request workflow described [here](./CONTRIBUTING-gitflow.md).
 
 ### <a name="commit-signing">Signing-off on Commits (Developer Certificate of Origin)</a>
 
 To contribute to this project, you must agree to the Developer Certificate of
 Origin (DCO) for each commit you make. The DCO is a simple statement that you,
-as a contributor, have the legal right to make the contribution.
+as a contributor, have the legal right to contribute.
```

Comment:
The original wording seems more appropriate here.

Reply:
Yes, this needs to stay as the original wording. The meaning changes under the new wording.